src/FluentMigrator.Runner.Postgres/Processors/Postgres/Postgres100Processor.cs

```diff
@@ -0,0 +1,40 @@
+#region License
+// Copyright (c) 2018, FluentMigrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using FluentMigrator.Runner.Generators.Postgres;
+using FluentMigrator.Runner.Initialization;
+
+using JetBrains.Annotations;
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace FluentMigrator.Runner.Processors.Postgres
+{
+    class Postgres100Processor : PostgresProcessor
+    {
+        public Postgres100Processor(
```

Comment:
Oops, I just noticed now all other processor classes override the DatabaseType and DatabaseAliases properties... while this appears to work without them, and I haven't looked into what those properties are used for yet (does anyone else know of the top of their heads?), perhaps I ought to similarly override them in this class?
https://github.com/fluentmigrator/fluentmigrator/blob/bfb074b3dd8158b721081950d44b5677f07500a6/src/FluentMigrator.Runner.Postgres/Processors/Postgres92/Postgres92Processor.cs#L34
https://github.com/fluentmigrator/fluentmigrator/blob/bfb074b3dd8158b721081950d44b5677f07500a6/src/FluentMigrator.Runner.Postgres/Processors/Postgres92/Postgres92Processor.cs#L36

Reply:
Upon closer inspection, these are what drive the `MigrationBase.IfDatabase` member.  For that reason I'll override them in the new `Postgres10_0Processor`.  I will specify PostgresSQL10_0 and PostgresSQL as alternate identifiers to the database tyoe Postgres10_0.

I also considered updating the existing `Postgres92Processor` to similarly specify `PostgreSQL` as an _additional_ alternate identifier (much like all the versions of SqlServer use plain `SqlServer` as an alternate so you can target any version by referencing this string without a version), so that it will match expressions that do `IfDatabase("PostgresSQL")` as v9.2 is still PostgresSQL.

Technically would be a breaking change though.  Up until now people could target only the first version of the processor by using `Postgres` or `PostgresSQL` without having 9.2 match... If we made this change then to avoid matching 9.2 they could only use `Postgres`.  Maybe it would be worth saving that change also for a 4.0 release where Postgres92 is renamed to Postgres9_2...?